.codecov.yml

```diff
@@ -1,3 +1,8 @@
 ignore:
   - "pyro/docutil.py"
   - "pyro/logger.py"
+
+coverage:
+  range: 60..95
```

Comment:
What does this range represent?

Reply:
how green that badge is :D

Reply:
Hahaha.. 👍

Reply:
i know what youre going to say but this is vital stuff :P

Reply:
To be honest..that light green badge was irking me a bit too.